RealEstate/RealEstate/Pages/Index.cshtml.cs

```diff
@@ -26,19 +26,38 @@ public void OnGet()
 
             using (var webClient = new WebClient())
             {
-                // Grab our JSON text. 
-                var houseJSON = webClient.DownloadString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+                
+                string Chicago_data = string.Empty;
+
+                try
+                {
+                    Chicago_data = webClient.DownloadString("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling API", e);
+                }
 
                 // Convert raw text to list.
-                List<Houses> houseCollection = Houses.FromJson(houseJSON);
+                List<Houses> houseCollection = Houses.FromJson(Chicago_data);
 
 
                 ViewData["Houses"] = houseCollection;
 
-                var landJSON = webClient.DownloadString("https://data.cityofchicago.org/resource/aksk-kvfp.json");
+                string Chicago_Land_data = string.Empty;
+
+                try
+                {
+                    Chicago_Land_data = webClient.DownloadString("https://data.cityofchicago.org/resource/aksk-kvfp.json");
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling API", e);
+                }
```

Comment:
Logging is a good idea, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That could be worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().